modules/samples/artifacts/SolrSample/SolrSample.siddhi

```diff
@@ -0,0 +1,93 @@
+@App:name("SolrSample")
```

Comment:
Can we change app name to store-solr.siddhi

Reply:
noted